src/status_im2/contexts/syncing/enter_sync_code/view.cljs

```diff
@@ -0,0 +1,81 @@
+(ns status-im2.contexts.syncing.enter-sync-code.view
+  (:require [clojure.string :as string]
+            [quo2.core :as quo]
+            [quo2.foundations.colors :as colors]
+            [reagent.core :as reagent]
+            [react-native.core :as rn]
+            [react-native.clipboard :as clipboard]
+            [utils.i18n :as i18n]
+            [status-im2.contexts.syncing.enter-sync-code.style :as style]
+            [utils.debounce :as debounce]
+            [utils.re-frame :as rf]
+            [status-im2.contexts.syncing.utils :as sync-utils]))
+
+(defn view
+  []
+  (let [default-value           (reagent/atom nil)
+        clipboard               (reagent/atom nil)
+        submit-button-disabled? (reagent/atom true)
+        profile-color           (rf/sub [:profile/customization-color])]
+    (fn []
+      (clipboard/get-string #(reset! clipboard %))
```

Comment:
could you elaborate on why effect in the render function?

Reply:
the idea was to get latest clipboard value when we reach this UI but now that I think about it, we could have done the same thing on-press event of paste button 🤔

Reply:
fixed : https://github.com/status-im/status-mobile/pull/16852/commits/4901f0af9aecacb16ab777747d3c959a744566a9

Reply:
thanks